preact-workerd-ssr-plugin/src/plugin.ts

```diff
@@ -43,16 +48,18 @@ export function preactWorkerdSSR() {
 
 						template = await server.transformIndexHtml(url, template);
 
-						const ssrResponse = await handler(req);
+						const { body } = await workerdFn({
+							entryPoint: "./entry-server.jsx",
+						});
```

Comment:
Here is the most important change, instead of having a handler that takes a request and returns a response we now have a function which takes serializable data and returns a serializable response, allowing us to be much more flexible